durabletask/worker.py

```diff
@@ -91,13 +92,15 @@ def __init__(self, *,
                  log_handler=None,
                  log_formatter: Optional[logging.Formatter] = None,
                  secure_channel: bool = False,
-                 interceptors: Optional[Sequence[shared.ClientInterceptor]] = None):
+                 interceptors: Optional[Sequence[shared.ClientInterceptor]] = None,
+                 max_workers: Optional[int] = None):
```

Comment:
The .NET SDK allows configuring separate maximum values for orchestrator work items, activity work items, and entity work items (see [here](https://github.com/microsoft/durabletask-dotnet/blob/b2f3fed1e741da67620104aed0eec340bd1b0425/src/Worker/Core/DurableTaskWorkerOptions.cs#L123-L131)). We need to do the same for the Python SDK for consistency. These values also need to be provided to the DTS backend as part of the `GetWorkItems` request, which I noted below.